src/js.rs

```diff
@@ -443,6 +443,16 @@ extern "C" {
     pub fn to_string(this: &Function) -> JsString;
 }
 
+impl JsValue {
+    pub fn as_function(&self) -> Option<Function> {
```

Comment:
Should this perhaps be `&Function` via a transmute? (same w/ `as_object` below)

Reply:
On it.